# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: b4b920f24bed
# Full Hash: b4b920f24bed3d02a7000892e9594c515970e1f6
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 3: Add mechanism to notify EarlyHintPreloader that HttpChannelParent is ready to complete the redirect r=necko-reviewers,kershaw,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D161178
# ==============================================================================

diff -r 13a9f4631369 -r b4b920f24bed netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Fri Dec 02 16:15:53 2022 +0000
+++ b/netwerk/protocol/http/moz.build	Fri Dec 02 16:15:54 2022 +0000
@@ -51,6 +51,7 @@
     "ClassifierDummyChannelParent.h",
     "ClassOfService.h",
     "EarlyHintPreloader.h",
+    "EarlyHintRegistrar.h",
     "EarlyHintsService.h",
     "HttpAuthUtils.h",
     "HttpBackgroundChannelChild.h",
@@ -103,6 +104,7 @@
     "ConnectionHandle.cpp",
     "DnsAndConnectSocket.cpp",
     "EarlyHintPreloader.cpp",
+    "EarlyHintRegistrar.cpp",
     "EarlyHintsService.cpp",
     "Http2Compression.cpp",
     "Http2Push.cpp",